apps/src/face_detection/openni_face_detection.cpp

```diff
@@ -19,7 +19,7 @@ run(pcl::RFFaceDetectorTrainer& fdrf, bool heat_map = false, bool show_votes = f
   OpenNIFrameSource::OpenNIFrameSource camera;
   OpenNIFrameSource::PointCloudPtr scene_vis;
 
-  pcl::visualization::PCLVisualizer vis("Face dection");
+  pcl::visualization::PCLVisualizer vis("Face detection");
```

Comment:
source change

Reply:
Thanks for the PR! Can you explain what you mean exactly with "source change"?

Reply:
This changes real source code (not documentation or comemnts), AFAICT

Reply:
Ah, got it, thanks!